editor/src/components/editor/actions/actions.tsx

```diff
@@ -4234,20 +4234,28 @@ export const UPDATE_FNS = {
         assertNever(textProp)
       }
     })()
-    const withCollapsedElements = collapseTextElements(action.target, withUpdatedText)
+    const withGroupTrueUpQueued: EditorState = {
```

Comment:
We do have `addToTrueUpGroups` for this!

Reply:
oh!